### ISSUE TITLE
#1772309941------------- Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#663]
<strong>Source:</strong>`<h5 class="baseH-sc-1pkt4xd-4 H5-sc-1pkt4xd-9 eepKhu jgksMb sc-fXynhf iCSxMR">Owner</h5>`
<strong>Element Location: </strong>.sc-GVOUr.iBLcpM:nth-child(3) > h5
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix any of the following:
  Element has insufficient color contrast of 3.03 (foreground color: #949494, background color: #ffffff, font size: 10.2pt (13.6px), font weight: normal). Expected contrast ratio of 4.5:1
Tags: cat.color,wcag2aa,wcag143,ACT
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
